src/mergeHead.ts

```diff
@@ -0,0 +1,64 @@
+import XXH from 'xxhashjs';
+
+interface Options {
+  space?: number | string;
+}
+
+const stringifyLD = (options: Options) => function () {
+  const jsonLd = this.$options.jsonld.call(this);
+
+  if (jsonLd !== null) {
+    const stringifiedJson = JSON.stringify(jsonLd, null, options.space);
+    const innerHTML = options.space === 0 ? stringifiedJson : `\n${stringifiedJson}\n`;
+
+    const hash = XXH.h32(innerHTML, 0).toString(16);
```

Comment:
Hashing is always cpu heavy. From the look of it you just need a unique variable here. You could also set an internal prop which you increment. 
```js
let jsonId = 1
...

const stringifyLD = (options: Options) => function () {
   const hash = jsonId++
...
```

Reply:
Thank you, here i copied the original implementation, but hashing is absolutely not needed. Removed and removed associated tests

Reply:
This approach of making `hid` changes `hid` by each request.
It causes problems when you make cache of HTML with ETag.
Please check this similar issue: https://github.com/ymmooot/nuxt-jsonld/issues/58. 🙏

Reply:
Uh, did not take account of this need. Wouldn't it be enough to use the name of the component that injected the jsonLD?

Reply:
Components can be used multiple time, so it will not be unique id.
Another approach that I wanted to try is gathering each json as an array and putting the array into root component. This approach does not need to make hid each json.

Anyway, you can just use hash just like before.
Avoiding using hash should be another issue.

Reply:
Yes, correct approach. Will put it back, so we can maybe close this issue :)